docs/docs/components.mdx

```diff
@@ -1445,9 +1445,7 @@ Intent classifiers assign one of the intents defined in the domain file to incom
   * `model_confidence`:
     This parameter allows the user to configure how confidences are computed during inference. It can take two values:
```

Comment:
Need a new intro sentence here

Reply:
@erohmensing Since the only non-deprecated option to use is `softmax` for now, how about removing this parameter from the list of recommended parameters to modify?

Reply:
Makes sense to me!

Reply:
Done! 🙏